lib/rubocop/target_finder.rb

```diff
@@ -175,7 +175,7 @@ def configured_include?(file)
     end
 
     def ruby_executable?(file)
-      return false unless File.extname(file).empty? && File.exist?(file)
+      return false unless File.extname(file).empty? && File.exist?(file) && !File.empty?(file)
 
       first_line = File.open(file, &:readline)
```

Comment:
`readline` raises but `gets` doesn't. I doubt its relevant here, just wanted to mention it.

```suggestion
      return false unless File.extname(file).empty? && File.exist?(file)
      return false unless (first_line = File.open(file, &:gets))

```

Reply:
Yeah, that was not clear for me that `readline` raises instead of returning `nil`. Perhaps that would be better to use `gets` here?